README.md

```diff
@@ -52,6 +52,8 @@ The webpack development server now uses https. If you're using a mac, you can si
 
 If you're using another operating system, you will need to ensure `mkcert` is installed, and then follow the instructions in the repository to generate certificates valid for `dev.new.expensify.com` and `localhost`. The certificate should be named `certificate.pem` and the key should be named `key.pem`. They should be placed in `config/webpack`.
 
+If you're using WSL or any other remote systems and trying to browse on your local computer, run `mkcert --install && mkcert -cert-file certificate.pem -key-file key.pem dev.new.expensify.com localhost 127.0.0.1` on your local and copy `certificate.pem` and `key.pem` to `config/webpack` folder manually, because `mkcert --install` command adds generated certs to the OS where you ran it.
```

Comment:
I think the previous paragraph is actually also meant to cover WSL and contains basically the same instructions as what you've added here, except it doesn't expand on what "the instructions in the repository" are. I assume this is the `mkcert --install && mkcert -cert-file certificate.pem -key-file key.pem dev.new.expensify.com localhost 127.0.0.1` step.

Reply:
Unless instructions in the repository refers to the mkcert repo? :sweat_smile:

Reply:
@jjcoffee I agree the previous paragraph is covering the WSL cases..., though what if change `If you're using another operating system` to  `If you're using another operating system or WSL`

Reply:
I think it's pretty clear without specifically mentioning WSL, let's close this PR for now. In future it's best if you raise issues on the Slack first in order to get consensus that it's worth making a change (that's why your issue was auto-closed).